proxies/live/apiproxy/policies/ServiceCallout.CallHealthcheckEndpoint.xml

```diff
@@ -13,7 +13,7 @@
         <LoadBalancer>
             <Server name="immunisation-history"/>
         </LoadBalancer>
-        <Path>/get</Path>
+        <Path>/immunisation-history/FHIR/R4/Immunization</Path>
```

Comment:
Is there a healthcheck endpoint on their backend instead?

Reply:
not yet